### DISCUSSION TITLE
Unable to paste into new page?

### DISCUSSION THREAD

**Discussion by shimizurei** [requarks/wiki#2856] [Help / Questions]
I'm testing this wiki out on Docker's Playground (https://labs.play-with-docker.com/).

I'm trying to simulate a move from Bookstack to WikiJS (my ultimate goal is to dump Bookstack for another wiki) by copy-pasting the text of one page into a page using the WYSIWYG editor. I can only paste one line at a time, which is tedious as heck. It couldn't even paste the bulleted list I copied. Is this expected behavior?

**Comment by NGPixel** [comment]
No, which browser are you using?

**Comment by shimizurei** [comment]
Chrome 87.0.4280.88 on a Win10 x64 machine.
